documentation/diagnostics-client-library-instructions.md

```diff
@@ -0,0 +1,522 @@
+# Microsoft.Diagnostics.NETCore.Client API Documentation 
+
+## Intro
+Microsoft.Diagnostics.NETCore.Client (also known as the Diagnostics Client library) is a managed library that lets you interact with .NET Core runtime (CoreCLR) for various diagnostics related tasks, such as tracing, requesting a dump, or attaching an ICorProfiler.
+
+## Installing 
+Microsoft.Diagnostics.NETCore.Client is available on [NuGet](https://www.nuget.org/packages/Microsoft.Diagnostics.NETCore.Client/). 
+
+## API Description
```

Comment:
I'd recommend putting conceptual information and small code samples earlier in documentation and API listings later (if at all). Ideally within the first page of text the reader can understand how the 'Hello World' scenario works. [Serilog](https://github.com/serilog/serilog/wiki/Getting-Started) shows a nice example.